assets/app/views/browse/build-config.html

```diff
@@ -28,6 +28,9 @@
                 ><i class="fa fa-play fa-fw" aria-hidden="true"></i> Start Build</a>
               </li>
               <li>
+                <a ng-href="{{buildConfig | editResourceURL}}" role="button"><i class="fa fa-pencil fa-fw" aria-hidden="true"></i> Edit </a>
```

Comment:
Does the "Edit" menu item look strange right above "Edit (Raw)"? Not sure what to suggest :(